Makefile

```
@@ -1,20 +1,23 @@
-.PHONY: help test lint markdownlint nixie fmt
+.PHONY: help test lint markdownlint nixie fmt typecheck
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)